stories/style.scss

```diff
@@ -1,3 +1,16 @@
 .greenDotLoader {
   background-color: #0f0 !important;
 }
+
+.toggle {
+  &[data-active="true"] {
```

Comment:
My only concern here is that we're exposing an implementation detail so any changes to data-... attributes could break things "upstream" in the consumers of these components. This might not necessarily be a bad thing if we consider data-* attributes (or all attributes) to be part of a public API. It's just that right now, afaik, the only API is component props and everything else (e.g. hooking onto classes or a specific structure of a component) should not be counted on.